scripts/add_webhook.sh

```diff
@@ -16,19 +16,24 @@ if [ -z "${GITHUB_SECRET_TOKEN}" ]; then
   exit 1
 fi
 
+if [ -z "${GITHUB_AUTH_TOKEN}" ]; then
```

Comment:
Oh, is this a separate token you made and configured? I don't know that I actually know the ocnfiguration behind the foremanbot account.